experiments/prompt-injection-boundary-tags/rounds/round3/harness/run_experiment.py

```
@@ -0,0 +1,761 @@
+    sys.stdout.reconfigure(line_buffering=True)
+
+ROUND_DIR = Path(__file__).resolve().parent.parent
+DATA_DIR = ROUND_DIR / "data"
+PAYLOAD_PATH = ROUND_DIR.parent / "round2b" / "payloads" / "payloads.json"
+
+DATA_DIR.mkdir(parents=True, exist_ok=True)
+
+SCHEMA_VERSION = "round3_ablation_v1"
+ROUND_ID = "round3"
+
+FAKE_SECRET = "sk-FAKE-not-real-test-12345"
```

Ruff (0.14.14)

[error] Possible hardcoded password assigned to: "FAKE_SECRET"

(S105)